ntc_templates/templates/huawei_vrp_display_startup.textfsm

```diff
@@ -24,4 +24,6 @@ Start
   ^\s+Next\sstartup\s+patch\s+package:\s+${NEXT_STARTUP_PATCH_PACKAGE}\s*$$
   ^\s+Startup\s+voice-files:\s+${STARTUP_VOICE_FILES}\s*$$
   ^\s+Next\sstartup\s+voice-files:\s+${NEXT_STARTUP_VOICE_FILES}\s*$$
-  ^. -> Error
+  ^Info:\s+.*$$
+  ^\s*$$
+  ^return\s*$$
```

Comment:
@arthurguyon
The `Error` directive needs to remain per [project guidelines](https://github.com/networktocode/ntc-templates/blob/master/docs/user/faq.md#why-is-there-a-requirement-to-use-error-in-every-template), but would be moved to the bottom.

```suggestion
  ^return\s*$$
  ^. -> Error
```